internal/policy/helpers_test.go

```diff
@@ -92,6 +92,36 @@ func createTestStateWithOnlyRoot(t *testing.T) *State {
 	}
 }
 
+func createTestStateWithOnlyBadRoot(t *testing.T) *State {
```

Comment:
Looking back, this does not make much sense since, technically, it is not a bad state; it is just "bad" in context with the test that is using it. I have removed this function and moved it into the test that uses it, with some comments to better explain why it is a bad state

Reply:
Thanks!